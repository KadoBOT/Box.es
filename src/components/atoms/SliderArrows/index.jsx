import React, {PropTypes} from 'react'

const SliderArrows = (props) => {
  return(
    <div>

    </div>
  )
}

SliderArrows.propTypes = {
  props: PropTypes.type
}

export default SliderArrows
