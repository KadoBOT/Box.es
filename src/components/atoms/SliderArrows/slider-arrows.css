.slider-arrows {
  position: fixed;
  bottom: 50%;
  z-index: 99;
  width: 100%;
  cursor: pointer;
}

.slider-arrows > .fa {
  text-shadow: 1px 1px 1px rgba(150, 150, 150, 1);
}

.slider-arrows > .fa-arrow-circle-right {
  position: absolute;
  right: 12.5vw;
  font-size: 85px;
  color: white;
  z-index: 99;
}

.slider-arrows > .fa-arrow-circle-left {
  position: absolute;
  left: -7.5vw;
  font-size: 85px;
  color: white;
  z-index: 99;
}
