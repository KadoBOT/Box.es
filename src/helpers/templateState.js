export default {
  "isFetching": false,
  "name": "Diamond Select Marvel Select: Juggernaut Action Figure",
  "description": "A Diamond Select Release! Sculpted by Gentle Giant! The ranks of the best-selling Marvel Select line expand to include one of the most deadly mutants ever, the unstoppable Juggernaut. Sculpted by Gentle Giant and standing a massive 7\" tall, Juggernaut is the latest in the, in scale, over sized figures from the Marvel Select line. Charles Xavier's half brother is ready to join Magneto in the fight against the X-Men and anyone else that stands in their way. Featuring Marvel Selects standard 16 points of articulation, Juggernaut, and the sans-helmet chase variant figure, are the textbook definition of Unstoppable!",
  "price": 22.43,
  "location_details": "Genosha",
  "currency": {
    "symbol": "$",
  },
  "likes_count": 80,
  "user": {
    "nickname": "xavier",
    "name": "Charles Xavier",
    "avatar": {
      "small_url": "http://static.comicvine.com/uploads/original/7/71941/1489898-charles_xavier.jpg",
    },
  },
  "item_photos": [
    {
      "large": "https://images-na.ssl-images-amazon.com/images/I/719022GksTL._SL1500_.jpg",
      "small": "https://images-na.ssl-images-amazon.com/images/I/719022GksTL._SL1500_.jpg",
      "id": 473580,
      "is_main": false
    },
    {
      "large": "http://www.angelfire.com/mech/jinsaotome/Blog/Juggyreview_001.JPG",
      "small": "http://www.angelfire.com/mech/jinsaotome/Blog/Juggyreview_001.JPG",
      "id": 473579,
      "is_main": false
    },
    {
      "large": "https://actionfigurecanada.files.wordpress.com/2011/09/juggernaut-marvel-select-figure-2.jpg",
      "small": "https://actionfigurecanada.files.wordpress.com/2011/09/juggernaut-marvel-select-figure-2.jpg",
      "id": 473581,
      "is_main": true
    },
    {
      "large": "http://4.bp.blogspot.com/-qNdk8wZtZSY/UgGmCHgBLJI/AAAAAAAB0pk/2qpwZTuNSls/s1600/52458-1.jpg",
      "small": "http://4.bp.blogspot.com/-qNdk8wZtZSY/UgGmCHgBLJI/AAAAAAAB0pk/2qpwZTuNSls/s1600/52458-1.jpg",
      "id": 473582,
      "is_main": false
    },
    {
      "large": "https://s-media-cache-ak0.pinimg.com/originals/08/38/8f/08388f9c134afd90f567be95a4fe610b.jpg",
      "small": "https://s-media-cache-ak0.pinimg.com/originals/08/38/8f/08388f9c134afd90f567be95a4fe610b.jpg",
      "id": 473583,
      "is_main": false
    }
  ],
  "recent_comments": [
    {
      "id": 184543,
      "comment": "Miss those classic outfits ",
      "user": {
        "id": 13118,
        "nickname": "colossus",
        "name": "Piotr Rasputin",
        "avatar": {
          "small_url": "http://cdn2-www.comingsoon.net/assets/uploads/2015/12/deadpool-colossus-header.jpg",
          "is_default": false
        },
        "share_url": "http://box.es/nixon"
      }
    },
  ],
}
