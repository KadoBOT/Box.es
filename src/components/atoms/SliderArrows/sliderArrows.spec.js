import React from 'react'
import { shallow, render, mount } from 'enzyme';

import templateState from '../../../helpers/_templateState'
import SliderArrows from './index'

it('should render the shallow', () => {
    const wrapper = shallow(
      <SliderArrows
        item={templateState.item}
        selectPicture={jest.fn()}
        selectedPicture={templateState.selectedPicture}
      />
    );
    expect(wrapper).toMatchSnapshot();
});
