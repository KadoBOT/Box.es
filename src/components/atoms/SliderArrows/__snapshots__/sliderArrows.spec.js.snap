exports[`test should render the shallow 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <div
    className="slider-arrows">
    <i
        className="fa fa-arrow-circle-left"
        onClick={[Function]} />
    <span />
    <i
        className="fa fa-arrow-circle-right"
        onClick={[Function]} />
</div>,
  "nodes": Array [
    <div
      className="slider-arrows">
      <i
            className="fa fa-arrow-circle-left"
            onClick={[Function]} />
      <span />
      <i
            className="fa fa-arrow-circle-right"
            onClick={[Function]} />
</div>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <Unknown
        item={
                Object {
                        "completeInfo": false,
                        "currency": Object {
                          "symbol": "$",
                        },
                        "description": "A Diamond Select Release! Sculpted by Gentle Giant! The ranks of the best-selling Marvel Select line expand to include one of the most deadly mutants ever, the unstoppable Juggernaut. Sculpted by Gentle Giant and standing a massive 7\" tall, Juggernaut is the latest in the, in scale, over sized figures from the Marvel Select line. Charles Xavier\'s half brother is ready to join Magneto in the fight against the X-Men and anyone else that stands in their way. Featuring Marvel Selects standard 16 points of articulation, Juggernaut, and the sans-helmet chase variant figure, are the textbook definition of Unstoppable!",
                        "item_photos": Array [
                          Object {
                            "id": 473580,
                            "is_main": false,
                            "large": "https://images-na.ssl-images-amazon.com/images/I/719022GksTL._SL1500_.jpg",
                            "small": "https://images-na.ssl-images-amazon.com/images/I/719022GksTL._SL1500_.jpg",
                          },
                          Object {
                            "id": 473579,
                            "is_main": false,
                            "large": "http://www.angelfire.com/mech/jinsaotome/Blog/Juggyreview_001.JPG",
                            "small": "http://www.angelfire.com/mech/jinsaotome/Blog/Juggyreview_001.JPG",
                          },
                          Object {
                            "id": 473581,
                            "is_main": true,
                            "large": "https://actionfigurecanada.files.wordpress.com/2011/09/juggernaut-marvel-select-figure-2.jpg",
                            "small": "https://actionfigurecanada.files.wordpress.com/2011/09/juggernaut-marvel-select-figure-2.jpg",
                          },
                          Object {
                            "id": 473582,
                            "is_main": false,
                            "large": "http://4.bp.blogspot.com/-qNdk8wZtZSY/UgGmCHgBLJI/AAAAAAAB0pk/2qpwZTuNSls/s1600/52458-1.jpg",
                            "small": "http://4.bp.blogspot.com/-qNdk8wZtZSY/UgGmCHgBLJI/AAAAAAAB0pk/2qpwZTuNSls/s1600/52458-1.jpg",
                          },
                          Object {
                            "id": 473583,
                            "is_main": false,
                            "large": "https://s-media-cache-ak0.pinimg.com/originals/08/38/8f/08388f9c134afd90f567be95a4fe610b.jpg",
                            "small": "https://s-media-cache-ak0.pinimg.com/originals/08/38/8f/08388f9c134afd90f567be95a4fe610b.jpg",
                          },
                        ],
                        "likes_count": 80,
                        "location_details": "Genosha",
                        "name": "Diamond Select Marvel Select: Juggernaut Action Figure",
                        "price": 22.43,
                        "user": Object {
                          "avatar": Object {
                            "small_url": "http://static.comicvine.com/uploads/original/7/71941/1489898-charles_xavier.jpg",
                          },
                          "name": "Charles Xavier",
                          "nickname": "xavier",
                          "share_url": "http://box.es/charlesxavier",
                        },
                      }
        }
        selectPicture={[Function]}
        selectedPicture={undefined} />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "item": Object {
            "completeInfo": false,
            "currency": Object {
              "symbol": "$",
            },
            "description": "A Diamond Select Release! Sculpted by Gentle Giant! The ranks of the best-selling Marvel Select line expand to include one of the most deadly mutants ever, the unstoppable Juggernaut. Sculpted by Gentle Giant and standing a massive 7\" tall, Juggernaut is the latest in the, in scale, over sized figures from the Marvel Select line. Charles Xavier\'s half brother is ready to join Magneto in the fight against the X-Men and anyone else that stands in their way. Featuring Marvel Selects standard 16 points of articulation, Juggernaut, and the sans-helmet chase variant figure, are the textbook definition of Unstoppable!",
            "item_photos": Array [
              Object {
                "id": 473580,
                "is_main": false,
                "large": "https://images-na.ssl-images-amazon.com/images/I/719022GksTL._SL1500_.jpg",
                "small": "https://images-na.ssl-images-amazon.com/images/I/719022GksTL._SL1500_.jpg",
              },
              Object {
                "id": 473579,
                "is_main": false,
                "large": "http://www.angelfire.com/mech/jinsaotome/Blog/Juggyreview_001.JPG",
                "small": "http://www.angelfire.com/mech/jinsaotome/Blog/Juggyreview_001.JPG",
              },
              Object {
                "id": 473581,
                "is_main": true,
                "large": "https://actionfigurecanada.files.wordpress.com/2011/09/juggernaut-marvel-select-figure-2.jpg",
                "small": "https://actionfigurecanada.files.wordpress.com/2011/09/juggernaut-marvel-select-figure-2.jpg",
              },
              Object {
                "id": 473582,
                "is_main": false,
                "large": "http://4.bp.blogspot.com/-qNdk8wZtZSY/UgGmCHgBLJI/AAAAAAAB0pk/2qpwZTuNSls/s1600/52458-1.jpg",
                "small": "http://4.bp.blogspot.com/-qNdk8wZtZSY/UgGmCHgBLJI/AAAAAAAB0pk/2qpwZTuNSls/s1600/52458-1.jpg",
              },
              Object {
                "id": 473583,
                "is_main": false,
                "large": "https://s-media-cache-ak0.pinimg.com/originals/08/38/8f/08388f9c134afd90f567be95a4fe610b.jpg",
                "small": "https://s-media-cache-ak0.pinimg.com/originals/08/38/8f/08388f9c134afd90f567be95a4fe610b.jpg",
              },
            ],
            "likes_count": 80,
            "location_details": "Genosha",
            "name": "Diamond Select Marvel Select: Juggernaut Action Figure",
            "price": 22.43,
            "user": Object {
              "avatar": Object {
                "small_url": "http://static.comicvine.com/uploads/original/7/71941/1489898-charles_xavier.jpg",
              },
              "name": "Charles Xavier",
              "nickname": "xavier",
              "share_url": "http://box.es/charlesxavier",
            },
          },
          "selectPicture": [Function],
          "selectedPicture": undefined,
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <div
          className="slider-arrows">
          <i
                    className="fa fa-arrow-circle-left"
                    onClick={[Function]} />
          <span />
          <i
                    className="fa fa-arrow-circle-right"
                    onClick={[Function]} />
</div>,
        "_debugID": 2,
        "_renderedOutput": <div
          className="slider-arrows">
          <i
                    className="fa fa-arrow-circle-left"
                    onClick={[Function]} />
          <span />
          <i
                    className="fa fa-arrow-circle-right"
                    onClick={[Function]} />
</div>,
      },
      "_renderedNodeType": 0,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <Unknown
    item={
        Object {
            "completeInfo": false,
            "currency": Object {
              "symbol": "$",
            },
            "description": "A Diamond Select Release! Sculpted by Gentle Giant! The ranks of the best-selling Marvel Select line expand to include one of the most deadly mutants ever, the unstoppable Juggernaut. Sculpted by Gentle Giant and standing a massive 7\" tall, Juggernaut is the latest in the, in scale, over sized figures from the Marvel Select line. Charles Xavier\'s half brother is ready to join Magneto in the fight against the X-Men and anyone else that stands in their way. Featuring Marvel Selects standard 16 points of articulation, Juggernaut, and the sans-helmet chase variant figure, are the textbook definition of Unstoppable!",
            "item_photos": Array [
              Object {
                "id": 473580,
                "is_main": false,
                "large": "https://images-na.ssl-images-amazon.com/images/I/719022GksTL._SL1500_.jpg",
                "small": "https://images-na.ssl-images-amazon.com/images/I/719022GksTL._SL1500_.jpg",
              },
              Object {
                "id": 473579,
                "is_main": false,
                "large": "http://www.angelfire.com/mech/jinsaotome/Blog/Juggyreview_001.JPG",
                "small": "http://www.angelfire.com/mech/jinsaotome/Blog/Juggyreview_001.JPG",
              },
              Object {
                "id": 473581,
                "is_main": true,
                "large": "https://actionfigurecanada.files.wordpress.com/2011/09/juggernaut-marvel-select-figure-2.jpg",
                "small": "https://actionfigurecanada.files.wordpress.com/2011/09/juggernaut-marvel-select-figure-2.jpg",
              },
              Object {
                "id": 473582,
                "is_main": false,
                "large": "http://4.bp.blogspot.com/-qNdk8wZtZSY/UgGmCHgBLJI/AAAAAAAB0pk/2qpwZTuNSls/s1600/52458-1.jpg",
                "small": "http://4.bp.blogspot.com/-qNdk8wZtZSY/UgGmCHgBLJI/AAAAAAAB0pk/2qpwZTuNSls/s1600/52458-1.jpg",
              },
              Object {
                "id": 473583,
                "is_main": false,
                "large": "https://s-media-cache-ak0.pinimg.com/originals/08/38/8f/08388f9c134afd90f567be95a4fe610b.jpg",
                "small": "https://s-media-cache-ak0.pinimg.com/originals/08/38/8f/08388f9c134afd90f567be95a4fe610b.jpg",
              },
            ],
            "likes_count": 80,
            "location_details": "Genosha",
            "name": "Diamond Select Marvel Select: Juggernaut Action Figure",
            "price": 22.43,
            "user": Object {
              "avatar": Object {
                "small_url": "http://static.comicvine.com/uploads/original/7/71941/1489898-charles_xavier.jpg",
              },
              "name": "Charles Xavier",
              "nickname": "xavier",
              "share_url": "http://box.es/charlesxavier",
            },
          }
    }
    selectPicture={[Function]}
    selectedPicture={undefined} />,
}
`;
